portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/PortalDatabaseValidationUpgrade.testcase

```diff
@@ -8,6 +8,54 @@ definition {
 	property portal.upstream = "true";
 	property testray.main.component.name = "Database Upgrade Framework";
 
+	@ignore = "true"
+	@priority = 4
+	test ComparePartitionedUpgradedAndFreshDBSchemas7413 {
+		property data.archive.type = "data-archive-portal-partition";
+		property database.partition.enabled = "true";
+		property liferay.online.properties = "true";
+		property portal.upgrades = "true";
+		property portal.version = "7.4.13.u33";
+
+		Portlet.shutdownServer();
+
+		AntCommands.runCommand("build-test.xml", "export-database-schema -Ddatabase.file.prefix=upgraded");
+
+		AntCommands.runCommand("build-test-local.xml", "database-reset");
```

Comment:
I added the use of this target since we needed to delete the previously generated partitions to then do the clean rebuild, I tried to do it through the root code but it didn't work, and this seemed like the best alternative, and it worked.